### PR TITLE
Travis integration

### PR DESCRIPTION
**Pull request by ddcprg**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -104,7 +104,91 @@
...
           </configuration>
         </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-release-plugin</artifactId>
+          <version>2.5.3</version>
+          <configuration>
+            <mavenExecutorId>forked-path</mavenExecutorId>
+            <useReleaseProfile>false</useReleaseProfile>
+            <arguments>${arguments}</arguments>
```
Comment (ddcprg) [inline pom.xml]: we can add `-Dmaven.deploy.skip=true -Dgpg.skip=true` here if we don't want to specify these 2 props in the CLI

File: travis/deploy.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+if [ "$TRAVIS_BRANCH" = 'master' ] && [ "$TRAVIS_PULL_REQUEST" == 'false' ]; then
+  if [ ! -z "$TRAVIS_TAG" ]; then
+    mvn clean deploy --settings travis/mvn-settings.xml -DskipTests=true -B -U -P oss-release "$@"
```
Comment (patduin) [inline travis/deploy.sh]: can the skipTests go in the mvn-settings.xml so we got all the stuff in one place?

File: pom.xml (hunk 1 above)
Comment (patduin) [inline pom.xml]: What CLI? I don't see them specified anywhere in the PR. Is the release plugin still used by Travis (or is this here for our Jenkins build)?

File: travis/deploy.sh (hunk 2 above)
Comment (ddcprg) [inline travis/deploy.sh]: sure

File: pom.xml (hunk 1 above)
Comment (ddcprg) [inline pom.xml]: when the release is done, either in CLI or in Jenkins

**Pull request merged**